x/params/handler.go

```diff
@@ -0,0 +1,58 @@
+package params
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/proposal"
+)
+
+func NewHandler(k ProposalKeeper) sdk.Handler {
+	return func(ctx sdk.Context, msg sdk.Msg) sdk.Result {
+		switch msg := msg.(type) {
+		case MsgSubmitParameterChangeProposal:
+			return handleMsgSubmitParameterChangeProposal(ctx, k, msg)
+		default:
+			errMsg := fmt.Sprintf("Unrecognized gov msg type: %T", msg)
+			return sdk.ErrUnknownRequest(errMsg).Result()
+		}
+	}
+}
+
+func handleMsgSubmitParameterChangeProposal(ctx sdk.Context, k ProposalKeeper, msg MsgSubmitParameterChangeProposal) sdk.Result {
+	content := NewProposalChange(msg.Title, msg.Description, msg.Changes)
+	return proposal.HandleSubmit(ctx, k.proposal, content, msg.Proposer, msg.InitialDeposit)
+}
+
+func NewProposalHandler(k ProposalKeeper) proposal.Handler {
+	return func(ctx sdk.Context, p proposal.Content) sdk.Error {
+		switch p := p.(type) {
+		case ProposalChange:
+			return handleProposalChange(ctx, k, p)
+		default:
+			// XXX
+			return nil
+		}
+	}
+}
+
+func handleProposalChange(ctx sdk.Context, k ProposalKeeper, p ProposalChange) (err sdk.Error) {
```

Comment:
```suggestion
func handleProposalChange(ctx sdk.Context, k ProposalKeeper, p ProposalChange) sdk.Error {
```

this way an `err` variable can be used within this func